CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 
 ## Unreleased
 
+### Fixed
+- `flyout` Always close flyout when clicking outside.
```

Comment:
I'd suggest talking with Jeroen Collier about the behaviour change - or wait for me to return to office ;-)

Reply:
Project is blocked on this bug. It's not a behavior change, it was supposed to close when you click outside the flyout, and this works for the flyout button, but it was broken for the datepicker's flyout because the document click event listener wasn't triggering angular's change detection.